apps/app_full.py

```diff
@@ -92,7 +92,7 @@ def update_controls():
     global scaler_crop
 
     # Fix aspect ratio of the pan/zoom
-    full_img = picam2.camera_properties["ScalerCropMaximum"]
```

Comment:
Actually (and sorry to be a pain...) could we do
```
_, full_img, _ = picam2.camera_controls['ScalerCrop']
```
I feel `camera_ctrl_info` should probably be made private at some point, and it would avoid people diving into the libcamera structures and having to do type conversions.